examples/dialogflow-entities-example/main.py

```diff
@@ -0,0 +1,226 @@
+            "entities": [{
+                "value": "@saving-account-types:saving-account-types",
+                "synonyms": ["@saving-account-types:saving-account-types"]
+            }, {
+                "value": "@checking-account-types:checking-account-types",
+                "synonyms": ["@checking-account-types:checking-account-types"]
+            }, {
+                "value":
+                    "@sys.date-period:date-period "
+                    "@saving-account-types:saving-account-types",
+                "synonyms": [
+                    "@sys.date-period:date-period "
+                    "@saving-account-types:saving-account-types"
+                ]
+            }, {
+                "value":
+                    "@sys.date-period:date-period "
+                    "@checking-account-types:checking-account-types",
+                "synonyms": [
+                    "@sys.date-period:date-period "
+                    "@checking-account-types:checking-account-types"
+                ]
+            }]
+        }]
+    }
+
+    client = get_dialogflow_client()
+    parent = get_agent(client)
+
+    # Uncomment if you want to create entities one by one.
+    # Please refer to README file for details.
+    # create_entities_type(client, entities, parent)
+
+    # Uncomment if you want to create in batch using entity_type_batch_inline.
+    # Please refer to README file for details.
+    # client.batch_update_entity_types(
+    #     parent=parent,
+    #     entity_type_batch_inline=entities_batch)
+
+    # Example how to create in batch using entity_type_batch_uri.
+    # entity_type_batch_uri is the URI to a Google Cloud Storage file containing
+    # entity types to update or create.
+    entity_type_batch_uri = "gs://<name of the bucket>/entities.json"
+    response = client.batch_update_entity_types(
+        parent=parent, entity_type_batch_uri=entity_type_batch_uri)
+
+    # This example uses futures for long-running operations returned from
+    # Google Cloud APIs.
+    # These futures are used asynchronously using callbacks and
+    # Operation.add_done_callback
+    # More info: https://googleapis.dev/python/google-api-core/1.14.3/futures.html
+    def callback(operation_future):
+        result = operation_future.result()
+
+    response.add_done_callback(callback)
+
+
+def create_entities_type(client, entities, parent):
+    """Creates entities.
+
+  Args:
+      client: dialogflow_v2.EntityTypesClient
+      entities: list of EntityTypes to create
+      parent: fully-qualified project_agent string
+  """
+    for entity_type in entities:
+        client.create_entity_type(parent, entity_type)
+
+
+def get_dialogflow_client():
+    """Returns the dialogflow entity types client."""
+
+    return dialogflow_v2.EntityTypesClient()
+
+
+def get_agent(client: dialogflow_v2.EntityTypesClient):
+    """Returns a fully-qualified project_agent string."""
+
+    return client.project_agent_path("<project_id>")
```

Comment:
This hard coding is not pretty.
Can this be either inferred from the GCF execution environment or the payload?